<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>analog_io</class>
 <widget class="QDialog" name="analog_io">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>381</width>
    <height>541</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>8</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Test Analog IO</string>
  </property>
  <widget class="QPushButton" name="okButton">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>500</y>
     <width>81</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>341</width>
     <height>101</height>
    </rect>
   </property>
   <property name="title">
    <string>Analog Output</string>
   </property>
   <widget class="QLineEdit" name="AO1Edt">
    <property name="geometry">
     <rect>
      <x>112</x>
      <y>30</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="AO2Edt">
    <property name="geometry">
     <rect>
      <x>112</x>
      <y>60</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="SetAnalogOutBtn">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>50</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Set Analog Out</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Pin 17 (AO1)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Pin 26 (AO2)</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>341</width>
     <height>101</height>
    </rect>
   </property>
   <property name="title">
    <string>Analog Input HD26-F connector</string>
   </property>
   <widget class="QLineEdit" name="AI2Edt">
    <property name="geometry">
     <rect>
      <x>112</x>
      <y>60</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="GetAnalogInBtn">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>40</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Get Analog In</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Pin 9 (AI1)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Pin 18 (AI2) </string>
    </property>
   </widget>
   <widget class="QLineEdit" name="AI1Edt">
    <property name="geometry">
     <rect>
      <x>112</x>
      <y>30</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>260</y>
     <width>341</width>
     <height>221</height>
    </rect>
   </property>
   <property name="title">
    <string>Onboard Analog Input</string>
   </property>
   <widget class="QLineEdit" name="ThermistorX11Edt">
    <property name="geometry">
     <rect>
      <x>112</x>
      <y>40</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="E1V2Edt">
    <property name="geometry">
     <rect>
      <x>112</x>
      <y>70</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="E5VIOEdt">
    <property name="geometry">
     <rect>
      <x>112</x>
      <y>100</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="E5VUSBEdt">
    <property name="geometry">
     <rect>
      <x>112</x>
      <y>130</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="NTC1X8Edt">
    <property name="geometry">
     <rect>
      <x>112</x>
      <y>160</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="NTC2X9Edt">
    <property name="geometry">
     <rect>
      <x>112</x>
      <y>190</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="GetOnboardAIBtn">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>110</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Get Onboard AI</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Thermistor X11</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>46</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string>1V2</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>46</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string>5VIO</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>46</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string>5VUSB</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>NTC1 X8</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>190</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>NTC2 X9</string>
    </property>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>AO1Edt</tabstop>
  <tabstop>AO2Edt</tabstop>
  <tabstop>SetAnalogOutBtn</tabstop>
  <tabstop>AI1Edt</tabstop>
  <tabstop>AI2Edt</tabstop>
  <tabstop>GetAnalogInBtn</tabstop>
  <tabstop>ThermistorX11Edt</tabstop>
  <tabstop>E1V2Edt</tabstop>
  <tabstop>E5VIOEdt</tabstop>
  <tabstop>E5VUSBEdt</tabstop>
  <tabstop>NTC1X8Edt</tabstop>
  <tabstop>NTC2X9Edt</tabstop>
  <tabstop>GetOnboardAIBtn</tabstop>
  <tabstop>okButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>okButton</sender>
   <signal>clicked()</signal>
   <receiver>analog_io</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>278</x>
     <y>253</y>
    </hint>
    <hint type="destinationlabel">
     <x>96</x>
     <y>254</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
